provider/aws/formation/resource/redis.json.tmpl

```diff
@@ -34,15 +37,31 @@
     }
   },
   "Outputs": {
-    "Url": { "Value": { "Fn::Sub": "redis://:${Password}@${ReplicationGroup.PrimaryEndPoint.Address}:${ReplicationGroup.PrimaryEndPoint.Port}/0" } }
+    "Url": {
+      "Value": {
+        "Fn::Sub": [
+          "${Protocol}://:${Password}@${ReplicationGroup.PrimaryEndPoint.Address}:${ReplicationGroup.PrimaryEndPoint.Port}/0",
```

Comment:
Should only include the Password in the URL if it's encrypted

Reply:
Ah thanks, missed that